scanomatic/ui_server_data/style/main.css

```diff
@@ -7,6 +7,13 @@
 input {
   padding: 3px;
   color: #333333;
+  background: #FFFFFF;
+}
+
+button {
```

Comment:
perhaps it makes sense to have a general button restyling but have you checked how this will look across _all_ views?

Reply:
not really sure how to check all views, so moved this to fixture.css